### ISSUE TITLE
Localstack container does not change region

### ISSUE THREAD

**Issue by SebastianStavar** [testcontainers/testcontainers-java#2408]
The localstack container allows to change env variables that localstack will consider. In my case I wanted to change DEFAULT_REGION to ap-southeast-2. But the getEndpointConfiguration() is hardcoded to use us-east-1 and is ignoring the meaning of  the env variable being changed. 

public AwsClientBuilder.EndpointConfiguration getEndpointConfiguration(Service service) should probably: 

return new AwsClientBuilder.EndpointConfiguration(
                "http://" +
                ipAddress +
                ":" +
                getMappedPort(service.getPort()), System.getProperty("DEFAULT_REGION", "us-east-1"));

 Tests still pass with the env variable set so I presume the aws client can still do things in a different region even if localstack default region is different, but it would be nice if they would be in sync.
 Regarding the above fix not sure if it requires supplementary validation (like what happens if an invalid region is set) or testcontainers is happy to let localstack deal with that.

**Comment by deroffal** [comment]
Please re-open.

It would be great if you could remove this hardcoded configuration, or at least update the doc 😉 

https://www.testcontainers.org/modules/localstack/
> Environment variables listed in Localstack's README may be used to customize Localstack's configuration. Use the .withEnv(key, value) method on LocalStackContainer to apply configuration settings.

**Comment by rnorth** [comment]
Reopening - sorry this was closed automatically.

**Comment by adrianpozueco** [comment]
@rnorth  I'm able to take a look to this one.
My idea is something like:
```
    public String getRegion() {
        return this.getEnvMap().containsKey("DEFAULT_REGION")?
            this.getEnvMap().get("DEFAULT_REGION")
            : DEFAULT_REGION;
    }
```
keeping current value into the constant.

If you are ok, I can open the PR

**Comment by eddumelendez** [comment]
It was fixed in 1.17.3.
